content/guides/figma/getting-started.mdx

```diff
@@ -54,3 +106,18 @@ To make components dynamic, we favor [component properties <LinkExternalIcon />]
 Whenever possible use components as they are without *detaching*.
 
 <a href="https://help.figma.com/hc/en-us/articles/360038662654-Guide-to-components-in-Figma">Learn more about Figma components <LinkExternalIcon /></a>
+
+## Using variable themes
+
+Primer Web provides *light mode* and *dark mode* using figma `variables`. This means you don't need a plugin to change between modes. Simply select any `frame` that uses `variables` or components with `variables`. Select the mode from the dropdown <svg class="svg" xmlns="http://www.w3.org/2000/svg" width="14" height="12" viewBox="0 0 14 12"><path fill="#000" fill-opacity="1" fill-rule="evenodd" d="m5 1.381-4 2.31v4.618l4 2.31 4-2.31V3.691L5 1.38zm5 1.732L5 .227 0 3.113v5.774l5 2.887 5-2.887V3.113zm3 5.196-5 2.887 1 .578 5-2.887V3.113L9 .227 8 .804l5 2.887v4.618zM6 6c0 .552-.448 1-1 1-.552 0-1-.448-1-1 0-.552.448-1 1-1 .552 0 1 .448 1 1z"></path></svg> in the *layer* section in the right sidebar
+
+<ImageBox caption="Switching the variable mode changes all nested items">
+  <img
+    width="800"
+    alt="Screen recording of switching variable modes in Figma"
+    src="https://github.com/primer/design/assets/813754/0675a32c-3c24-40a7-a030-4c60e1b0409f"
+  />
+</ImageBox>
+
+### Video introduction to primer web variables and modes / themes (github internal)
+<a href="https://github.rewatch.com/video/qvryg0cscoowmpyx-figma-primer-web-update-to-figma-variables"><img src="https://thumbnails.rewatch.com/thumbs/qvryg0cscoowmpyx?fm=jpg&p=true&q=100" /></a>
```

Comment:
Note: Remember that this is only for internal use. As I mentioned before, we need to make it clear that it's for internal purposes. If we decide to make the Figma library public in the future, maybe we could download the video and host it on a public URL?

Reply:
It is publicly available forever already. 😉